backend/api/v1/endpoints/Research/Collaboration.py

```
@@ -0,0 +1,143 @@
+        new_request = CollaborationRequest(
+            research_id=research_id,
+            requester_id=current_user.id,
+            message=message
+        )
+        db.add(new_request)
+        db.commit()
+        db.refresh(new_request)
+
+        return {"message": "Collaboration request sent successfully"}
+
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)